src/client.test.ts

```diff
@@ -1,40 +1,58 @@
 import Client = require('./client')
 import nock = require('nock')
+import EventSource from 'eventsource'
+
+jest.mock('eventsource', () => {
+  return jest.fn().mockImplementation((urlArg) => {
+    return {
+      addEventListener: jest.fn(),
+      url: urlArg
+    }
+  })
+})
```

Comment:
Alternately,
```typescript
jest.mock('eventsource', () => jest.fn().mockImplementation((urlArg) => {
  addEventListener: jest.fn(),
  url: urlArg
}))
```

Also, is this code supposed to be using semicolon statement terminators?

Reply:
Not sure, but `npm run build` fails with this change about an expected semi-colon which doesn't make sense to me.

Reply:
It's probably water under the bridge now, but I expect the problem is the abiguity of the open brace:  it probably needs to be put inside parentheses in order to make it clear that it is an object value and not a code block.  (My bad; again, apologies for the misdirection.)
```typescript
jest.mock('eventsource', () => jest.fn().mockImplementation((urlArg) => ({
  addEventListener: jest.fn(),
  url: urlArg
)}))
```